src/components/workspace/WorkspaceSidebar.svelte

```diff
@@ -0,0 +1,344 @@
+              <Button builders={[builder]} variant="ghost">
+                <Files size={16} />
+              </Button>
+            </Tabs.Trigger>
+          </Tooltip.Trigger>
+          <Tooltip.Content sideOffset={8}>
+            <div>Files</div>
+          </Tooltip.Content>
+        </Tooltip.Root>
+        <Tooltip.Root>
+          <Tooltip.Trigger asChild let:builder>
+            <Tabs.Trigger value="grid" class="flex h-10 w-10 items-center justify-center rounded-none shadow-none">
+              <Button builders={[builder]} variant="ghost">
+                <FolderTree size={16} />
+              </Button>
+            </Tabs.Trigger>
+          </Tooltip.Trigger>
+          <Tooltip.Content sideOffset={8}>
+            <div>Grid</div>
+          </Tooltip.Content>
+        </Tooltip.Root>
+        <Tooltip.Root>
+          <Tooltip.Trigger asChild let:builder>
+            <Button class="h-10" builders={[builder]} variant="ghost" on:click={onActionsClick}>
+              <Clapperboard size={16} />
+            </Button>
+          </Tooltip.Trigger>
+          <Tooltip.Content sideOffset={8}>
+            <div>Actions</div>
+          </Tooltip.Content>
+        </Tooltip.Root>
+        <!-- <Tooltip.Root>
+          <Tooltip.Trigger asChild let:builder>
+            <Tabs.Trigger value="settings" class="flex h-10 w-10 items-center justify-center rounded-none shadow-none">
+              <Button builders={[builder]} variant="ghost">
+                <Settings size={16} />
+              </Button>
+            </Tabs.Trigger>
+          </Tooltip.Trigger>
+          <Tooltip.Content sideOffset={8}>
+            <div>Settings</div>
+          </Tooltip.Content>
+        </Tooltip.Root> -->
+      </Tabs.List>
+    </div>
+    <div class="flex h-full w-full flex-col">
+      <Tabs.Content value="files" class="mt-0 h-full">
+        <div class="grid h-full grid-rows-[min-content_auto]">
+          <Sidebar.Header className="p-0">
+            <div class="flex items-center justify-between gap-0 border-b border-border bg-background p-[8px]">
+              <SectionTitle>Workspace Tree View</SectionTitle>
+              <div class="flex gap-0">
+                <Tooltip.Root>
+                  <Tooltip.Trigger asChild let:builder>
+                    {#if didWorkspaceUpdate}
+                      <Button builders={[builder]} variant="ghost">
+                        <Check size={16} />
+                      </Button>
+                    {:else}
+                      <Button builders={[builder]} variant="ghost" on:click={onRefreshWorkspace}>
+                        <RefreshCw size={16} />
+                      </Button>
+                    {/if}
+                  </Tooltip.Trigger>
+                  <Tooltip.Content>
+                    <div class="text-sm">Refresh (last {getTimeAgo(lastRefreshTime, new Date())})</div>
```

Comment:
The copy here feels a little terse - could it be something more like "Last refreshed X seconds ago"?
<img width="189" height="91" alt="image" src="https://github.com/user-attachments/assets/54e17573-50bc-4f62-965f-0311ac14087e" />

Reply:
I'm okay with that! @lklyne came up with it so I'll give him a chance to give a final 👍🏻 on this.